### ISSUE TITLE
 Installation Error: @storybook/addon-a11y@npm:6.5.26: No candidates found

### ISSUE THREAD

**Issue by Akshay-Patel-dev** [RocketChat/Rocket.Chat#31357]
I'm encountering a package installation error when running the yarn install command.

Details of the Issue:

Description:

The yarn install command fails due to a versioning problem with the @storybook/addon-a11y package.
The specific error message is: @storybook/addon-a11y@npm:6.5.26: No candidates found.

Steps to Reproduce:
1. Clone the repository.
2. Change the Node version to 14.
3. Install Meteor version 2.13.3.
4. Run yarn install.

Expected Behavior:
The yarn install command should successfully install all required packages.

Actual Behavior:
The command fails with the aforementioned error.
Additional Information:

I've attached a screenshot of the error message for your reference (see attached image).
![image](https://github.com/RocketChat/Rocket.Chat/assets/95700402/13112092-4144-480b-b729-5396f9a960cf)

I've attempted troubleshooting steps such as verifying package names and versions, clearing the Yarn cache, and checking for network connectivity issues, but the error persists.

I have observed that the specified version of @storybook/addon-a11y is listed as 6.5.26, which, upon closer examination, appears to be unavailable within the @storybook/addon-a11y package.

To provide clarity, I am outlining the versions that are currently accessible for @storybook/addon-a11y within the range of 6.5.13 to 7.0.0-alpha.1:

 ['6.5.13',
  '6.5.14-alpha.0',
  '6.5.14-alpha.1',
  '6.5.14-alpha.2',
  '6.5.14',
  '6.5.15-alpha.0',
  '6.5.15-alpha.1',
  '6.5.15',
  '6.5.16-alpha.0',
  '6.5.16',
  '6.5.17-alpha.0',
  '7.0.0-alpha.0',
  '7.0.0-alpha.1',]

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest version and it was fixed [here](https://github.com/RocketChat/Rocket.Chat/commit/3ebc8e0868c859d2d8e636787645c29a89dea1e5)

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
